src/api.js

```diff
@@ -26,4 +26,18 @@ export function getArticle(id) {
   });
 }
 
+export function getCommentsByArticleId(id) {
+  return fetch(
+    `https://news-webpage-project.onrender.com/api/articles/${id}/comments`
```

Comment:
recommend refactoring to use axios just because it handles things like queries much nicer than fetch which you'll have to look at using later on